toxygen/main.py

```diff
@@ -59,11 +59,6 @@ def main(self):
         if platform.system() == 'Linux':
             QtCore.QCoreApplication.setAttribute(QtCore.Qt.AA_X11InitThreads)
 
-        # application color scheme
-        with open(curr_directory() + '/styles/style.qss') as fl:
```

Comment:
I think we can load dark style here. Now login screen has light style but dark style login screen is definitely better